code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -381,6 +381,25 @@
 	beakers += B2
 
 
+/obj/item/weapon/grenade/chem_grenade/ez_clean
+	name = "cleaner grenade"
+	desc = "Waffle Co.-brand foaming space cleaner. In a special applicator for rapid cleaning of wide areas."
+	stage = READY
+
+/obj/item/weapon/grenade/chem_grenade/ez_clean/New()
+	..()
+	var/obj/item/weapon/reagent_containers/glass/beaker/large/B1 = new(src)
+	var/obj/item/weapon/reagent_containers/glass/beaker/large/B2 = new(src)
+
+	B1.reagents.add_reagent("fluorosurfactant", 40)
+	B2.reagents.add_reagent("water", 40)
+	B2.reagents.add_reagent("ez_clean", 60) //ensures a  t h i c c  distribution

```

Comment:
To keep the functionality that makes it look the exact same and still work as a cleaner, yes